src/amoc_dist.erl

```diff
@@ -31,6 +32,12 @@ ping_nodes() ->
 do(Scenario, Start, End) ->
     do(Scenario, Start, End, []).
 
+-spec test_status(atom()) -> amoc_controller:scenario_status().
+test_status(ScenarioName) ->
```

Comment:
When calling function with non-existing scenario as parameter it returns 'loaded'.